internal/controller/amaltheasession_controller.go

```diff
@@ -144,7 +144,18 @@ func (r *AmaltheaSessionReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 
 	log.Info("spec", "cr", amaltheasession)
 
-	children := NewChildResources(amaltheasession)
+	children, err := NewChildResources(amaltheasession)
+	if err != nil {
+		log.Error(
+			err,
+			"Could not template child resources",
```

Comment:
I am not sure how to interpret that message.

Reply:
I will revise and clarify it.